src/main/java/lt/andriaus/hangman/util/StringUtils.java

```diff
@@ -0,0 +1,38 @@
+package lt.andriaus.hangman.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class StringUtils {
+
+    static public Set<Character> stringToCharSet(String inputString) {
+        return inputString.chars()
+                .mapToObj(e -> (char) e)
+                .collect(Collectors.toSet());
+    }
+
+    static public List<Character> stringToCharList(String inputString) {
+        return inputString.chars()
+                .mapToObj(e -> (char) e)
+                .collect(Collectors.toList());
+    }
+
+    static public String charSetToString(Set<Character> inputSet) {
+        return inputSet.stream()
+                .map(String::valueOf)
+                .collect(Collectors.joining(""));
+    }
+
+    static public List<String> getRegexGroups(String regexPattern, String string) {
```

Comment:
are u sure, kad nori su regexu? niekas jo nemėgsta ir beveik viską gali padaryt paprasčiau be regexo :D